src/snapred/backend/recipe/algorithm/__init__.py

```diff
@@ -12,13 +12,16 @@ def loadModule(x):
     from mantid.api import AlgorithmFactory
     from mantid.simpleapi import _create_algorithm_function
 
-    while True:
+    for i in range(30):
```

Comment:
could you clarify the why `30` is the desired range in a comment?

Reply:
It was just a good guess to try alot but without infitnite loops.  Thinking on it, the best number is the length of the module list.